plugin-server/tests/worker/ingestion/person-state.test.ts

```diff
@@ -190,14 +195,17 @@ describe('PersonState.update()', () => {
     })
 
     it('updates person properties if needed', async () => {
-        await hub.db.createPerson(timestamp, { b: 3, c: 4 }, {}, {}, 2, null, false, uuid.toString(), ['new-user'])
+        await hub.db.createPerson(timestamp, { b: 3, c: 4, d: 7 }, {}, {}, 2, null, false, uuid.toString(), [
+            'new-user',
+        ])
 
         const personContainer = await personState({
             event: '$pageview',
             distinct_id: 'new-user',
             properties: {
                 $set_once: { c: 3, e: 4 },
                 $set: { b: 4 },
+                $unset: ['d'],
```

Comment:
Can we extract this to separate test? 

This test is now testing too many subtle behaviors at once.